andela/asgard-cfh
# [feature]: Implement crown levels to encourage user donations

### What does this PR do?
- This Pull request adds the feature for a user to be motivated with a crown to encourage donation.
### Description of Task to be completed?
- The aim is to Define alternatives for increasing donations received through the app.
- A user gets a silver or gold crown depending on the amount donated.
### How should this be manually tested?
- start the app: `npm run start:dev`
- Check for user image while game starts. There should be a silver crown, gold crown or no crown.
### Any background context you want to provide?
- N/A
### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/157801306
### Screenshots (if appropriate)
- No
### Questions:
- No
